Libraries/Components/ScrollResponder.js

```diff
@@ -476,7 +476,7 @@ var ScrollResponderMixin = {
    * you had explicitly focused a node etc).
    */
   scrollResponderKeyboardWillShow: function(e: Event) {
-    this.keyboardWillOpenTo = e;
+    if (e.endCoordinates) this.keyboardWillOpenTo = e;
```

Comment:
Please follow the style consistent with this file and put curly braces around all if blocks.